### ISSUE TITLE
Error code 41

### ISSUE THREAD

**Issue by ram1123** [tompollard/phd_thesis_markdown#51]
Hi

While make I am getting following error:

pandoc "/tmp/phd_thesis_markdown/source"/*.md \
-o "/tmp/phd_thesis_markdown/output/thesis.pdf" \
-H "/tmp/phd_thesis_markdown/style/preamble.tex" \
--template="/tmp/phd_thesis_markdown/style/template.tex" \
--bibliography="/tmp/phd_thesis_markdown/source/references.bib" 2>pandoc.log \
--csl="/tmp/phd_thesis_markdown/style/ref_format.csl" \
--highlight-style pygments \
-V fontsize=12pt \
-V papersize=a4paper \
-V documentclass:report \
-N \
--latex-engine=xelatex 
Makefile:28: recipe for target 'pdf' failed
make: *** [pdf] Error 41

Please let me know what I am doing wrong.

My system is Ubuntu 16.04.

Let me know if you need more information.

with regards,
Ram

**Comment by tompollard** [comment]
Have you installed the required packages? If so, are you able the run the following command?:

```pandoc "/tmp/phd_thesis_markdown/source/09_chapter_1.md" -o "/tmp/phd_thesis_markdown/output/test.pdf"```

**Comment by ram1123** [comment]
Yes, I am able to run this command.

> ramkrishna@ramkrishna-Dell-System-XPS-L502X:/tmp/phd_thesis_markdown$ ls output/
> source  thesis.html  thesis.pdf
> ramkrishna@ramkrishna-Dell-System-XPS-L502X:/tmp/phd_thesis_markdown$ pandoc "/tmp/phd_thesis_markdown/source/09_chapter_1.md" -o "/tmp/phd_thesis_markdown/output/test.pdf"
> ramkrishna@ramkrishna-Dell-System-XPS-L502X:/tmp/phd_thesis_markdown$ ls output/
> source  test.pdf  thesis.html  thesis.pdf

**Comment by ram1123** [comment]
Hi,

One more information. I checked now then 

=> make html
and
=> make  docx

are working fine. 

Also, 

=> make tex 

is successfully running and  it is generating the tex file. but I am not able to run it to produce pdf.

**Comment by ram1123** [comment]
Hi,

any suggestions about this?

with regards,
Ram

**Comment by Nowosad** [comment]
Have you tried to build a pdf based on the generated tex file (in texmaker, texstudio, etc?)?

**Comment by ram1123** [comment]
Yes I tried it running using **pdflatex** command but its not working.

**Comment by Nowosad** [comment]
Did you get any error messages?

**Comment by ram1123** [comment]
Error message:

Package ctable Warning: 
(ctable)                Transparency disabled: pdfTeX is not running in PDF mod
e .

) (/usr/share/texlive/texmf-dist/tex/latex/morefloats/morefloats.sty)
(/usr/share/texlive/texmf-dist/tex/xelatex/xunicode/xunicode.sty

! LaTeX Error: *** this package currently works only with XeTeX ***
 You are not using XeTeX, so we are exiting cleanly!
 Continuing may lead to 'unavailable font metrics' errors!
.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.113 ...sname\relax\expandafter\notXeTeXformat\fi
                                                  
? 
) (/usr/share/texlive/texmf-dist/tex/latex/xltxtra/xltxtra.sty
! 
 ********************************************
 * XeTeX is required to compile this document.
 * Sorry!
 ********************************************.
\RequireXeTeX ...********************************}
                                                  \endgroup \fi 
l.26 \RequireXeTeX
                  
? 
(/usr/share/texlive/texmf-dist/tex/latex/fontspec/fontspec.sty
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/expl3.sty
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/expl3-code.tex)
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/l3dvips.def))
(/usr/share/texlive/texmf-dist/tex/latex/l3packages/xparse/xparse.sty)

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Fatal fontspec error: "cannot-use-pdftex"
! 
! The fontspec package requires either XeTeX or LuaTeX.
! 
! You must change your typesetting engine to, e.g., "xelatex" or
! "lualatex"instead of plain "latex" or "pdflatex".
! 
! See the fontspec documentation for further information.
! 
! For immediate help type H <return>.
!...............................................  
                                                  
l.28 \msg_fatal:nn {fontspec} {cannot-use-pdftex}
                                                 
? 
 ) ) )
(\end occurred when \ifx on line 99 was incomplete)
No pages of output.
Transcript written on thesis.log.

**Comment by Nowosad** [comment]
It looks like you should use **xelatex** insted of **pdflatex**. Could you try it and put error messages here.

**Comment by ram1123** [comment]
Hi,

It gives me one error:

> ! LaTeX Error: File `siunitx.sty' not found.

But, I got the pdf file and it looks fine.

**Comment by Nowosad** [comment]
Try to install `siunitx.sty` and `make pdf`. Right now it suppose to work.

**Comment by ram1123** [comment]
I installed it and first did

> make tex
> cd output
> xelatex thesis.tex

It worked fine without any error.

But when I tried running:

> make pdf

then I get following error:

pandoc "/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/source"/*.md \
-o "/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/output/thesis.pdf" \
-H "/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/style/preamble.tex" \
--template="/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/style/template.tex" \
--bibliography="/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/source/references.bib" 2>pandoc.log \
--csl="/home/ramkrishna/PhD_New_Dir_16July2016/MyWritings/phd_thesis_markdown/style/ref_format.csl" \
--highlight-style pygments \
-V fontsize=12pt \
-V papersize=a4paper \
-V documentclass:report \
-N \
--latex-engine=xelatex 
Makefile:28: recipe for target 'pdf' failed
make: *** [pdf] Error 43
